engine/services/model_service.cc

```diff
@@ -195,13 +194,27 @@ cpp::result<std::string, std::string> ModelService::HandleUrl(
   auto file_name{url_obj.pathParams.back()};
 
   if (author == "cortexso") {
-    // TODO: try to get the branch
     return DownloadModelFromCortexso(model_id);
   }
 
+  if (url_obj.pathParams.size() < 5) {
```

Comment:
Please add comments

Reply:
Sorry, I missed this one. I will add comment in the following PR.